static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -171,6 +172,14 @@ const show = (): Promise<boolean> => import(
             appboy.logInAppMessageImpression(messageConfig);
         }
 
+        // Log the impression with Ophan
+        submitViewEvent({
+            component: {
+                componentType: 'BRAZE_BANNER',
+                id: messageConfig.extras.componentName,
```

Comment:
For now this will only ever be one value: DigitalSubscriberAppBanner since that's the only component we currently support.